andela/AndelaEatsUI
# #165903299 Implement rating an unordered meal

#### What does this PR do?
-Implements rating a meal from a previous day's menu if a user had not ordered a meal
#### Description of Task to be completed?
- Fetch past days menu
- Render meals or orders to rate conditionally
#### How should this be manually tested?
- Log into the application If you don't have any orders placed you'll see meals to rate from the past two days menus if you have an order placed delete it to and you'll see meals from the past days, select any meals and rate.
#### What are the relevant pivotal tracker stories?
- [#165903299d](https://www.pivotaltracker.com/story/show/165903299)

#### Background Context
N/A
#### Screenshots (If Applicable)
<img width="824" alt="Screenshot 2019-05-14 at 21 26 49" src="https://user-images.githubusercontent.com/39955261/57722409-09e64e80-768f-11e9-9231-6b207a8d19f1.png">
